.travis.yml

```diff
@@ -17,18 +17,17 @@ env:
   matrix:
     - PLONE_VERSION=4.3
     - PLONE_VERSION=4.3 DEXTERITY_ONLY=true
-    - PLONE_VERSION=4.3 JQUERY=1.9.1.3
+    - PLONE_VERSION=4.3 VERSIONS=versions:plone.app.jquery=1.9.2
```

Comment:
you can pass any part/directive value on command line to override those on configuration; that's on buildout documentation:

https://travis-ci.org/collective/collective.cover/jobs/245957084#L1805